assets/tasks/common.json

```diff
@@ -132,31 +132,6 @@
                 "group": "cloud-related"
             }
         },
-        {
-            "label": "set-platform-fleet-name",
-            "detail": "",
```

Comment:
this actually is a util task, it was set a hide true wrongly, maybe because was not description field.

Reply:
Sure, I will fix it

Reply:
This is being covered by the fill-pipeline-settings task now I believe